src/nsidc/metgen/metgen.py

```diff
@@ -732,6 +732,20 @@ def create_ummg(configuration: config.Config, granule: Granule) -> Granule:
         configuration.ummg_path(), granule.producer_granule_id
     )
 
+    # get premet if it exists
+    if granule.premet_filename == "":
+        raise Exception(
+            f"premet_dir {granule.premet_filename} is specified but no premet file exists for granule."
+        )
+    premet_content = utilities.premet_values(granule.premet_filename)
+
+    # temporarily show any additional attributes in premet file
+    if "AdditionalAttributes" in premet_content:
```

Comment:
This output will be removed once the code is in place to write the additional attributes to the UMM-G file.